packages/oidc-server/src/salt-service.ts

```diff
@@ -35,18 +38,19 @@ export default defineEventHandler(async (event) => {
       audience: APP_AUD,
     });
 
-    const iss = payload.iss;
-    const aud = payload.aud;
-    const sub = payload.sub;
+    const iss = payload.iss as string;
```

Comment:
If this can be undefined we should handle this. If it cannot be we can use zod that it's already included to do a nice parse of the payload object.